src/gui.rs

```diff
@@ -107,54 +93,56 @@ enum Interaction {
     SelectInstallation(Installation),
     SelectLoaderVersion(LoaderVersion),
     SelectMcVersion(MinecraftVersion),
-    EnableSnapshots(bool),
-    EnableBetas(bool),
+    ToggleSnapshots(bool),
+    ToggleBetas(bool),
```

Comment:
The way I see it, this message triggers whenever the checkbox is toggled ('set' makes sense too). I personally found naming this 'enable' really confusing when I first started working on this project because that implies the message only triggers when the checkbox is enabled.

Reply:
I would accept `set`. I don't think `toggle` is guaranteed. if it was, we wouldn't need the boolean argument.

Reply:
Is `SetShowSnapshots` too long? Would `SetSnapshot` is descriptive enough?

Reply:
I originally wanted to suggest `SetSnapshotsEnabled`, but I already expected you didn't like `EnableSnapshots` and wouldn't like that either.
`SetShowSnapshots` accurately describes what it does, especially if you rename trhe state variable to `show_snapshots`.

Reply:
fe9321182e3ccbbe916b52da0f71e91f0b813051